people/.htaccess

```diff
@@ -4,6 +4,7 @@ RewriteRule ^agbeltran$ http://newt.oerc.ox.ac.uk/people/agb.rdf [R=303,L]
 RewriteRule ^aidan$ http://aidan.droppages.com/ [R=302,L]
 RewriteRule ^amy$ http://bosatsu.net/foaf/uberconf/amy.rdf [R=303,L]
 RewriteRule ^asinesio$ http://asinesio.github.io/foaf.rdf [R=303,L]
+RewriteRule ^bgilstrap$ http://bosatsu.net/foaf/brian.rdf://dl.dropboxusercontent.com/u/3128671/brian.html [R=303,L]
```

Comment:
Did you mean this to be https://bosatsu.net/foaf/brian.rdf://dl.dropboxusercontent.com/u/3128671/brian.html?